homeTask_4/homeTask_4/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+
+namespace homeTask_4
+{
+    class Program
+    {
+
+        
+//1. Создать два произвольных массива, подсчитать максимальное и минимальное значение их элементов.Вывести на экран.
+//2. Поменять в массивах максимальные и минимальные значения местами
+        static void InitializeArray(int[] array)
+        {
+            Random rnd = new Random();
+            for (int i = 0; i < array.Length; ++i)
+            {
+                array[i] = rnd.Next(1, 50);
+            }
+        }
+
+        static void PrintArray(int[] array)
+        {
+            for (int i = 0; i < array.Length; ++i)
+            {
+                Console.WriteLine("№={0} is {1}", i + 1, array[i]);
+            }
+        }
+
+        static int MaxElement(int[] array)
+        {
+            int max = array[0];
+            for (int i = 0; i < array.Length; ++i)
+                if (array[i] > max) 
+                 max = array[i];
+                    
+            return max;
+        }
+
+        static int MinElement(int[] array)
+        {
+            int min = array[0];
+            for (int i = 0; i < array.Length; i++)
+                if (array[i] < min)
+                   min = array[i];
+                 
+            return min;
+        }
+        static void SwitchArray(int[] array)
+        {
+            int min = array[0];
+            int max = array[0];
+            int minNumber = 0;
+            int maxNumber = 0;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] > max) {
```

Comment:
what will be if array[i] == max or min ?

Reply:
No problem. Yes, I know about this case, but I thought it is not important within this task. I will do this.

Reply:
Yep, it's not important in this case. But we learn hot to write best of the best code =)
So, let's try to write best code rigth now. If you start it from simple tasks you wiil do it on your future projects.

Reply:
@EgorAntonovich I've done it and resolved the conversation.